aiounifi/controller.py

```diff
@@ -179,7 +188,7 @@ def message_handler(self, message: dict) -> dict:
                 if res.content_type == "application/json":
                     response = await res.json()
                     _raise_on_error(response)
-                    return response["data"]
+                    return response.get("data") or response
```

Comment:
I noticed not every request included a `data` object in the response, but expected the entire response instead.

Reply:
This didn't work for me since it didn't cover the case when the data value is an empty list, but using parts of it. thanks